From 856238c37ff156f2b5fea89f04d4e4f954da071f Mon Sep 17 00:00:00 2001
From: Lionel Xu <Lionel.Xu@freescale.com>
Date: Fri, 29 Jun 2012 12:50:32 +0800
Subject: ENGR00176555 ESAI: Increase DMA buffer size to reslove occasional
 aplay underrun

The underrun warning appears when playback high bit-rate and multi-channel(
greater than 6) wav, increase DMA buffer size can resolve this issue.

Signed-off-by: Lionel Xu <Lionel.Xu@freescale.com>
---
 sound/soc/imx/imx-ssi.h |    5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/sound/soc/imx/imx-ssi.h b/sound/soc/imx/imx-ssi.h
index 1b43c62..c57f9fe 100644
--- a/sound/soc/imx/imx-ssi.h
+++ b/sound/soc/imx/imx-ssi.h
@@ -223,9 +223,6 @@ int imx_pcm_new(struct snd_card *card, struct snd_soc_dai *dai,
 	struct snd_pcm *pcm);
 void imx_pcm_free(struct snd_pcm *pcm);
 
-/*
- * Do not change this as the FIQ handler depends on this size
- */
-#define IMX_SSI_DMABUF_SIZE	(64 * 1024)
+#define IMX_SSI_DMABUF_SIZE	(256 * 1024)
 
 #endif /* _IMX_SSI_H */
-- 
1.7.9.5

